README.md

```diff
@@ -1 +1,9 @@
 # Delivery Service
+
+##### GitHub workflow
+1. Clone this repo to your machine
+2. `cd` into repository and `git checkout develop`
+3. Create new branch for your feature, e.g. `git checkout -b DDD-14-Implement-ui`
+4. After done with your work, push your local branch to remote, e.g. `git push -u origin DDD-14-Implement-ui`
```

Comment:
I guess we could add sample commit message too, f.e.: `git commit -m "DDD-14 Change this and that"`